apps/chat/src/components/Common/Buttons/ToggleSidebarButtor.tsx

```diff
@@ -33,26 +35,35 @@ export const ToggleSidebarButton: React.FC<Props> = ({
     onToggle();
   };
 
+  const ToggleIcon = useCallback(() => {
+    const iconProps = {
+      className: 'text-secondary hover:text-accent-primary',
+      width: iconSize,
+      height: iconSize,
+    };
+
+    if (isOpened)
+      return rightSide ? (
+        <MoveRightIcon {...iconProps} />
+      ) : (
+        <MoveLeftIcon {...iconProps} />
+      );
+
+    return rightSide ? (
+      <MoveLeftIcon {...iconProps} />
+    ) : (
+      <MoveRightIcon {...iconProps} />
+    );
+  }, [iconSize, isOpened, rightSide]);
+
   return (
     <Tooltip isTriggerClickable tooltip={t(tooltip)}>
       <button
         className="flex h-full items-center justify-center px-3 md:px-5"
         data-qa={dataQa}
         onClick={handleToggle}
       >
-        {isOpened ? (
-          <MoveLeftIcon
-            className="text-secondary hover:text-accent-primary"
-            width={iconSize}
-            height={iconSize}
-          />
-        ) : (
-          <MoveRightIcon
-            className="text-secondary hover:text-accent-primary"
-            width={iconSize}
-            height={iconSize}
-          />
-        )}
+        <ToggleIcon />
```

Comment:
need to use `rotate-180` for right side
```suggestion
        <Icon
          className={classNames(
            'text-secondary  hover:text-accent-primary',
            rightSide && 'rotate-180',
          )}
          width={iconSize}
          height={iconSize}
        />
```